torch/_inductor/codecache.py

```diff
@@ -2948,12 +2948,20 @@ def _codegen_glue(cls, argtypes, headerfile):
     @classmethod
     @functools.lru_cache(None)
     def config_hash(cls):
+        from torch._inductor.cpp_builder import CppBuilder, CppOptions
+
+        dummy_builder = CppBuilder(
```

Comment:
Nit: can we use a different name other than `dummy_builder`?

Reply:
@desertfire Done, change it to `command_gen`.